<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetServoParamsPassword</class>
 <widget class="QWidget" name="WidgetServoParamsPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;
border-color:#FAFDFF;
border: 1px solid #DFE3E5;
border-radius: 4px;
color:#2D3440;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>42</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelRandomNum">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
border:none;</string>
       </property>
       <property name="text">
        <string>随机码：</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="linePassword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #37475f;
border: 1.2px solid #C6C9D7;
border-radius: 4px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="labelConfirmStatus">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnConfirm">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  width: 196px;
  height: 44px;
  background: #0047BB;
  border-radius: 4px;
  color: #FAFDFF;
}
QPushButton::disabled{
  background: rgb(55, 71, 95);
}</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>42</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QPushButton" name="btnClose">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/image/images/icon_close_dialog.png);
color: gray;
}

QPushButton:hover{
border-image: url(:/image/images/icon_close_hover.png);

}

QPushButton:pressed{
border-image: url(:/image/images/icon_close_press.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
